src/Console/Commands/StaticWarm.php

```diff
@@ -170,6 +171,15 @@ private function uris(): Collection
             ->values();
     }
 
+    protected function shouldVerifySsl(): bool
+    {
+        if ($this->option('insecure')) {
+            return false;
+        }
+
+        return ! $this->laravel->isLocal();
```

Comment:
This can be simplified to a one-liner as so:
```php
return ! ($this->option('insecure') || $this->laravel->isLocal());
```

Reply:
Yeah I told him to use a method. The one liner is confusing.